.pre-commit-config.yaml

```
@@ -58,7 +58,7 @@ repos:
     hooks:
       - id: interrogate
         name: Run interrogate (linter for docstrings)
-        args: [--fail-under=95, --ignore-init-module, --style=google, -vv, meshpy/]
+        args: [--fail-under=100, --ignore-init-module, --style=google, -vv, meshpy/]
```

If we want to also check the docstring coverage within the tests we should probably also add the path here?

See https://github.com/davidrudlstorfer/pygebege/blob/ffcb1e370bb087f2bf189bce0e376a7a77e00fdb/.pre-commit-config.yaml#L23

---

I think that is a private repository, the link does not work for me.

---

Sorry my fault - https://github.com/davidrudlstorfer/pyskel/blob/a929e2b8f849fb84d19d3a7b9dbd7e575a72d446/.pre-commit-config.yaml#L48

This should work

```suggestion
        args: [--fail-under=100, --ignore-init-module, --style=google, -vv, meshpy/, tests/]
```

Basically that was my idea. I think we also need the path to the test folder if the docstring coverage should be checked?

---

Sounds good - if it works out of the box let's drop the path arguments 🤝

---

I tried, it works, a wrong doctoring cause the pipeline to fail. The current state should be good to go.

---

Thanks for testing the missing docstring 🙏🏻